quickstart.md

```diff
@@ -324,7 +324,12 @@ The image encryption key and key for SSH access have been attached to the CoCo s
 
 To learn more about creating custom policies, see the section on [Creating a simple-kbs Policy to Verify the SEV Firmware Measurement](#creating-a-simple-kbs-policy-to-verify-the-sev-firmware-measurement).
 
-Currently, the SEV unencrypted image use case also requires the `simple-kbs` to be deployed. This will change in a future CoCo release.
+A KBS is not required to run unencrypted containers.
+Instead, disable pre-attestation by editing the Kata config file located at `/opt/confidential-containers/share/defaults/kata-containers/configuration-qemu-sev.toml`.
+```
+guest_pre_attestation = false
```

Comment:
@fitzthum, I'd avoid recommending users to directly edit something in the node.  The main reason for that being that the Operator may end up re-writting that file at any point in time, leading to a non-expected / non-functional state.

Reply:
The SEV stuff already requires the user to update the config file to specify the location of the KBS among other things. Will the operator really just replace this file at random? That's not ideal. The plan is to have most of these values set through Kubernetes annotations, but that hasn't been tested out for this release.

Reply:
Then I'd leave this out from this release.

Reply:
Well we definitely need at least part of the message telling people they don't need the kbs for unencrypted images. I guess we could leave out the details of how they actually do this, but that seems a bit silly. Like I said we already have other pieces that rely on updating the config. What are the details of the operator replacing the config file?

Reply:
There are also general Kata features that depend on config file changes. For instance a user might want to update the guest memory size via the config file. If these kind of changes don't persist as expected, I think we have a bug.

Reply:
We need ways to expose some configurations to be set / defined by the Operator, indeed.  That's the case of the AA_KBC, that may be the case for some of the Kata Containers configurations, including the one you mentioned.

I don't consider this a bug, just something was not implemented yet.  Please, mind to open a RFE on the Operator to track this for the coming releases?

Reply:
If the K8s node restarts, or the operator pod restarts, then the files will be replaced with the ones shipped with the runtime payload used by the operator.
You can probably add a warning and mention that for having a different kata configuration (other than the shipped version), a new runtime payload with the updated kata configuration file should be used.

Going forward we can think about exposing kata configuration values via the operator and avoid rebuilding payload for config change. Also the annotation work will help.